src/nvim/tui/input.c

```diff
@@ -199,18 +199,24 @@ static void forward_mouse_event(TermInput *input, TermKeyKey *key)
     len += (size_t)snprintf(buf + len, sizeof(buf) - len, "Right");
   }
 
-  if (ev == TERMKEY_MOUSE_PRESS) {
+  switch (ev) {
+  case TERMKEY_MOUSE_PRESS:
     if (button == 4) {
       len += (size_t)snprintf(buf + len, sizeof(buf) - len, "ScrollWheelUp");
     } else if (button == 5) {
       len += (size_t)snprintf(buf + len, sizeof(buf) - len, "ScrollWheelDown");
     } else {
       len += (size_t)snprintf(buf + len, sizeof(buf) - len, "Mouse");
     }
-  } else if (ev == TERMKEY_MOUSE_DRAG) {
+    break;
+  case TERMKEY_MOUSE_DRAG:
     len += (size_t)snprintf(buf + len, sizeof(buf) - len, "Drag");
-  } else if (ev == TERMKEY_MOUSE_RELEASE) {
+    break;
+  case TERMKEY_MOUSE_RELEASE:
     len += (size_t)snprintf(buf + len, sizeof(buf) - len, "Release");
+    break;
+  default:
```

Comment:
Don’t use `default` for enums. And `case`s need to be indented, they are only not indented in legacy code.